### PR TITLE
Add Qt

### PR DESCRIPTION
**Pull request by soodaayush**
Quick question - Since the name of the website is less than three characters (which is the minimum character limit when searching for websites), do we still keep the name as Qt or name it to something else?

### CODE REVIEW

**Comment by soodaayush** [comment]
Weird, CircleCI workflow is saying qt.io doesn't meet Similarweb rankings, but it does here: [https://www.similarweb.com/website/qt.io/#overview](https://www.similarweb.com/website/qt.io/#overview)

**Comment by Carlgo11** [comment]
For me it looks like all CircleCI checks were successful.

**Comment by soodaayush** [comment]
It initially failed, when I made the pull request, guess it was a bug.

File: img/q/qt.io.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...-6.7 6.67H0V12.976L6.597 6.4z" fill="#41cd52"/><path d="M12.714 13.8c1.572-1.04 3.516-1.303 5.364-1.286 1.76.022 3.607.33 5.066 1.38 1.06.745 1.75 1.895 2.208 3.085.743 1.953.96 4.062.986 6.138.003 2.1-.1 4.226-.783 6.218a6.96 6.96 0 0 1-2.51 3.393l2.53 4.06-3.155 1.457-2.7-4.44c-1.217.2-2.46.184-3.684.05-1.465-.176-2.958-.622-4.088-1.608-.87-.752-1.414-1.805-1.787-2.875-.6-1.8-.775-3.736-.803-5.635-.015-2.053.116-4.138.736-6.108.48-1.482 1.272-2.96 2.6-3.83zm17.846-.066h3.304l.002 4.37 4.223.001-.077 2.836c-1.407.003-2.812-.001-4.22.002l.005 7.163c.027.654.012 1.343.3 1.947.2.465.775.585 1.235.6l2.602-.07.154 2.607c-1.726.3-3.537.637-5.25.095-.722-.214-1.395-.693-1.688-1.407-.525-1.215-.567-2.566-.6-3.87v-7.065c-.768-.002-1.536.001-2.303-.002v-2.835l2.302-.002.002-4.37z" fill="#fff"/><path d="M16.676 15.5c1.07-.14 2.19-.102 3.203.302a3.49 3.49 0 0 1 1.884 1.763c.564 1.22.78 2.564.89 3.893.118 1.768.102 3.553-.142 5.31-.17 1.046-.416 2.15-1.124 2.974-.72.828-1.828 1.183-2.89 1.266-1.122.082-2.305 0-3.318-.53-.59-.315-1.093-.808-1.386-1.413-.355-.724-.55-1.515-.688-2.306-.414-2.5-.513-5.108.052-7.603.233-.94.57-1.908 1.26-2.617.598-.608 1.432-.922 2.263-1.04z" fill="#41cd52"/></svg>
```
Comment (Carlgo11) [inline img/q/qt.io.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 35.22"><path fill="#41cd52" d="M6.6 0H48v28.55l-6.7 6.67H0V6.58Z"/><path fill="#fff" d="M18.08 6.1c-1.85 0-3.79.26-5.37 1.3h-.02c-1.32.87-2.12 2.35-2.6 3.83a19.65 19.65 0 0 0-.73 6.11c.03 1.9.2 3.83.8 5.63a6.58 6.58 0 0 0 1.78 2.88 7.5 7.5 0 0 0 4.1 1.61c1.22.13 2.46.15 3.68-.05l2.7 4.44 3.15-1.46-2.52-4.06a6.96 6.96 0 0 0 2.5-3.4 19 19 0 0 0 .79-6.2 18 18 0 0 0-.99-6.15 6.55 6.55 0 0 0-2.2-3.09c-1.47-1.05-3.31-1.35-5.07-1.38zm12.48 1.23v4.39h-2.3v2.83h2.3v7.07a10.32 10.32 0 0 0 .6 3.87 2.6 2.6 0 0 0 1.69 1.41c1.71.54 3.52.2 5.25-.1l-.16-2.61-2.6.07c-.46-.01-1.04-.13-1.24-.6-.28-.6-.27-1.29-.3-1.94v-7.17H38l.09-2.85h-4.23V7.33h-3.3zm-12.8 1.69a6.41 6.41 0 0 1 2.12.38 3.49 3.49 0 0 1 1.88 1.76c.57 1.22.78 2.57.9 3.9.11 1.77.1 3.55-.15 5.3a5.9 5.9 0 0 1-1.12 2.98 4.2 4.2 0 0 1-2.9 1.27c-1.11.08-2.3 0-3.31-.53-.6-.32-1.1-.81-1.4-1.42a8.54 8.54 0 0 1-.67-2.3c-.42-2.5-.52-5.1.05-7.6.23-.94.57-1.9 1.26-2.62.6-.6 1.43-.92 2.26-1.04a6.41 6.41 0 0 1 1.08-.08z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Changes made.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**